api/v1alpha1/volumesnapshotrestore_types.go

```diff
@@ -37,6 +37,8 @@ type VolumeSnapshotRestoreStatus struct {
 	Phase VolumeSnapshotRestorePhase `json:"phase,omitempty"`
 	// name of the volumesnapshot snaphandle that is backed up
 	SnapshotHandle string `json:"snapshotHandle,omitempty"`
+	// name of the volumesnapshot that is backed up
+	SnapshotName string `json:"snapshotName,omitempty"`
```

Comment:
Shouldn't this be `VolumeSnapshotContent` Name ? instead of `SnapshotName`, might mean VS instead of VSC.